pkg/minikube/node/cache.go

```diff
@@ -107,7 +108,15 @@ func beginDownloadKicArtifacts(g *errgroup.Group, driver string, cRuntime string
 			out.T(out.Pulling, "Pulling base image ...")
 			g.Go(func() error {
 				glog.Infof("Downloading %s to local daemon", baseImage)
```

Comment:
Please add a TODO to respect `--image-repository`: https://github.com/kubernetes/minikube/issues/7915